### PR TITLE
Update docs for new page URL features

### PR DESCRIPTION
**Pull request by WimJongeneel**
The new features for page URLs are to be released in Mendix 10.0.0 Beta 2. Related to [UFC-696].

### CODE REVIEW

File: content/en/docs/refguide/modeling/pages/page/page-properties.md [hunk 1]
```diff
@@ -77,17 +77,19 @@ The title can be overridden. For example, the [Create button](/refguide/control-
...
+Pages with parameters can also have URLs. For those pages all page parameters should be present in the URL. This is done by defining which attribute of each page parameter should be used in the URL. The syntax for this is the name of the page parameter followed by a `/` and then the name of the attribute that should be used, placed between curly brackets. For example, in the URL `/product/{Product/Name}` the `Name` attribute of the page parameter `Product` will be used in the URL (in a browser, the URL will look like `http://example.mendixcloud.com/p/product/hammer`). Any attribute of type `Boolean`, `Decimal`, `Enumeration`, `Integer`, `Long` or `String` can be used in the URL. Next to this `Id` can be used as an attribute to include the identifier of the entity in the URL, for example: `/product/{Product/Id}`.
 
 {{% alert color="warning" %}}
-Page URLs are not supported for pages with multiple parameters.
+Page URLs are not supported for pages with NPE parameters
```
Comment (Robinfr) [inline content/en/docs/refguide/modeling/pages/page/page-properties.md]: Should be written out, e.g.:

```suggestion
Page URLs are not supported for pages that have non-persistable entities as parameters
```

**Review by Robinfr — CHANGES_REQUESTED** [review]
I would like to see atleast one screenshot around configuring the page URL.

**Review by Robinfr — APPROVED** [review]
Can be checked by tech writer @Adam-Dupaski

**Comment by ConnorLand** [comment]
Hello. I talked with Jonathan Diehl today about upcoming improvements to Page URLs and Deeplinking. I see this PR is slated for Beta 2. 

How big of a portion of the Page URL updates does this PR capture? Will there be bigger Page URL PRs/updates for GA?

**Comment by Robinfr** [comment]
Hi @ConnorLand it's likely there will be more changes for Page URLs and other related features for the docs for GA. We want to get these docs out there, as we want people to start playing around with the feature in beta 2.

**Review by ConnorLand — APPROVED** [review]

**Pull request merged**